Documentation update for page parameters [WTF-2288] (#10472)
mendix/docs: content/en/docs/refguide/modeling/pages/page/page-properties.md

#### Parameters {#parameters}

> The list of [parameters](/refguide/page-parameter/) that this page expects. These parameters can be used by [data views](/refguide/data-view/) with a page parameter data source.

Reviewer: ```suggestion
The list of [parameters](/refguide/page-parameter/) that this page expects. These parameters can be used by widgets on the page.
```
